scripts/ezfio_interface/qp_convert_output_to_ezfio.py

```diff
@@ -421,3 +444,4 @@ def get_full_path(file_path):
         print file_, 'recognized as', str(res_file).split('.')[-1].split()[0]
 
     write_ezfio(res_file, ezfio_file)
+    os.system("qp_run save_ortho_mos "+ezfio_file)
```

Comment:
Yes, but if we forget to do this step, all the QP after will be wrong.
I added a warning message in case it does not work.